portality/api/v2/crud/articles.py

```diff
@@ -83,6 +84,9 @@ def create(cls, data, account):
             raise Api400Error("; ".join(e.errors))
         except DuplicateArticleException as e:
             raise Api403Error(str(e))
+        except IngestException as e:
+            raise Api400Error(str(e) if str(e) else e.message)
```

Comment:
this is a distressing mix of python2 and python3 exceptions - but I guess it's ok if that's where we're at.

Reply:
I don't like this part neither. But for some reason the Ingest Exeption that is raised here returns `' '` for `str(e)` but has `message` attribute.